apiserver/src/main/java/org/dependencytrack/resources/v1/AccessControlResource.java

```diff
@@ -132,7 +132,7 @@ public Response retrieveProjects(@Parameter(description = "The UUID of the team
     public Response retrieveUserProjects(
             @Parameter(description = "The username to retrieve projects for", required = true) @PathParam("username") String username) {
 
-        try (QueryManager qm = new QueryManager()) {
+        try (QueryManager qm = new QueryManager(getAlpineRequest())) {
```

Comment:
Ensuring that `request` is available for all persistence operations across all resources. Only then can permission and access control checks be performed effectively.